File: 2020/J2/J2.cpp
#include <bits/stdc++.h>

using namespace std;

int P, N, R;

int main() {
    scanf("%i\n%i\n%i", &P, &N, &R);
    int infected = N; int people = N;
    int count = 0;
    while (true) {
        people = people + (infected*R);
        infected = infected*R;
        count++;
        if (people >= P) {
            if (people == P) {
                printf("%i\n", count+1);
            } else {
                printf("%i\n", count);
            }
            break;
        }
    }
}

File: 2020/J2.py
"""
author: sjay05
"""
P = input()
N = input()
R = input()

infected = N
people = N

count = 0
while True:
    people = people + (infected*R)
    infected = infected*R
    count += 1
    if people >= P:
        if people == P:
            print count+1
        else:
            print count
        break
